packages/sirius-web/frontend/sirius-web-application/src/views/explorer/ExplorerView.tsx

```diff
@@ -136,11 +147,12 @@ export const ExplorerView = ({ editingContextId, readOnly }: WorkbenchViewCompon
           editingContextId={editingContextId}
           readOnly={readOnly}
           treeId={'explorer://'}
+          tree={tree}
```

Comment:
Here you are giving a `GQLTree | null` to TreeView which expects a `GQLTree`. On the other hand, in `ModelBrowserTree` you are not rendering `TreeView` unless you have a `GQLTree`. So who is responsible for testing that `tree` is `null`?

Given the fact that both `ModelBrowser` is testing for `null` and the type of `tree` in `TreeViewProps` is `GQLTree`, I suspect that `ExplorerView` should do the job.